src/realm.js

```diff
@@ -97,7 +97,13 @@ export type PropertyBindings = Map<PropertyBinding, void | Descriptor>;
 export type CreatedObjects = Set<ObjectValue>;
 export type CreatedAbstracts = Set<AbstractValue>;
 
-export type SideEffectType = "MODIFIED_BINDING" | "MODIFIED_PROPERTY" | "EXCEPTION_THROWN" | "MODIFIED_GLOBAL";
+export type SideEffectType = "MODIFIED_BINDING" | "MODIFIED_PROPERTY" | "MODIFIED_GLOBAL";
```

Comment:
Why is "EXCPETION_THROWN" no longer possible?

Reply:
We removed it ages ago and I forgot to change it here. Throwing isn't considered a side-effect in a pure function anymore.